lib/rubygem_fs.rb

```diff
@@ -70,13 +70,22 @@ def get(key)
       nil
     end
 
+    # Util method used for debugging and console scripts
+    def deleted?(key)
```

Comment:
this, actually wont tell if the file is deleted. a `NoSuchKey` could mean a deleted file or a file that never existed.